pre_commit/repository.py

```diff
@@ -190,6 +191,21 @@ def _cloned_repository_hooks(repo_config, store, root_config):
     )
 
 
+_TOP_LEVEL_KEYS = ('repo', 'rev', 'hooks')
```

Comment:
ideally this should be derived from `CONFIG_REPO_SCHEMA` in `pre_commit.clientlib` -- it probably also makes sense to implement this check as a `cfgv` checker in that module

also these aren't top level keys, just repository-level keys